aroquev/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,35 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> comments;
+
+  @Override
+  public void init() {
```

Comment:
+1. I think it'll make more sense when something is adding comments, but this would probably be the best way to do it.

Reply:
Mmm, like this?
``` java
private static final ArrayList<String> comments = new ArrayList<String>() {{
    comments.add("I would like to live in NYC.");
    comments.add("Berlin looks like a nice place to have fun.");
    comments.add("I gotta go visit Japan one day!");
  }};  
```

Reply:
I found it a bit weird that it did not allow me to do this:
``` java
private static final ArrayList<String> comments = new ArrayList<String>();
comments.add("I would like to live in NYC.");
comments.add("Berlin looks like a nice place to have fun.");
comments.add("I gotta go visit Japan one day!");
```
Any idea why? I think because I'm calling a method outside a method (the constructor in the commet that does work)?

Reply:
Right. You can't really do anything outside of methods or functions in the class in Java. I think this kind of has the different ways you could do this in an initializer: https://howtodoinjava.com/java/collections/arraylist/initialize-arraylist/